x-pack/plugins/apm/public/components/app/service_groups/refresh_service_groups_subscriber.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useRef } from 'react';
+import { Subject, Subscription } from 'rxjs';
+
+const refreshServiceGroupsSubject = new Subject<any[]>();
+
+export function refreshServiceGroups(...args: any[]) {
```

Comment:
Shouldn't this at minimum be `unknown` instead of `any`? Any way we can narrow it more?
```suggestion
export function refreshServiceGroups(...args: unknown[]) {
```

Reply:
thanks, your right, but now i think of it, i never really pass in anything in the args, so i could just omit the parameter definition entirely to make it simpler.